main.rb

```diff
@@ -0,0 +1,30 @@
+require './classes/app'
+def show_options
+  options = [
+    '1) List all books',
+    '2) List all music albums',
+    '3) List all movies',
+    '4) List of games',
+    "5) List all genres (e.g 'Comedy', 'Thriller')",
+    "6) List all labels (e.g. 'Gift', 'New')",
+    "7) List all sources (e.g. 'From a friend', 'Online shop')",
+    '8) Add a book',
+    '9) Add a music album',
+    '10) Add a movie',
+    '11) Add a game',
+    '12) List all authors',
```

Comment:
[OPTIONAL] It would be a great idea if you put all options in group order. In your options array I can see one of the option is out of its placement and missing some parts you can follow this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things_divided.md#group-task:~:text=List%20all%20authors%20(e.g.%20%27Stephen%20King%27)) to see.